src/web/layouts/InteractiveLayout.tsx

```diff
@@ -0,0 +1,734 @@
+import React from 'react';
```

Comment:
Ditto https://github.com/guardian/dotcom-rendering/pull/2991/files#r631828742

Reply:
Ah thanks - actually I should remove this from this PR! Originally, I planned to implement both new layouts here, but then decided against to keep the PR smaller.